bridge/device/cloud/manager.go

```diff
@@ -430,7 +433,7 @@ func (c *Manager) signOff(ctx context.Context) error {
 	}
 	// signIn / refresh token fails
 	if ctx.Err() != nil {
-		return nil
```

Comment:
@jkralik linter complained about ctx.Err not being nil, but nil being returned. Was that deliberate?

Reply:
this is fine